src/collect-adb.rst

```diff
@@ -1,108 +1,205 @@
+__ https://www.howtogeek.com/125769/how-to-install-and-use-abd-the-android-debug-bridge-utility_
 
-To install :command:`adb` as a standalone tool, please follow the instructions given `here <https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/>`_.
+.. _install-adb-standalone:
 
-Forms can be manipulated from the command line itself. The following sections describe how :command:`adb` can be used to work with the app.
+Standalone ADB
+~~~~~~~~~~~~~~~~
+
+You can also `install ADB without Android studio`__.
+This is not well supported, though,
+and should only be done
+if you cannot install Android Studio on your computer.
+
+__ https://android.gadgethacks.com/how-to/android-basics-install-adb-fastboot-mac-linux-windows-0164225/_
+
+.. _adb-forms:
+
+Managing Forms with ADB
+---------------------------
+
+Forms are stored on the device in  :file:`sdcard/odk/forms/`.
 
 .. _loading-blank-forms-with-adb:
 
 Loading blank forms
 ~~~~~~~~~~~~~~~~~~~~
 
-The forms are stored in :file:`sdcard/odk/forms/` folder on the device. They can be loaded via a USB device using:
-
 .. code-block:: console
 
-  $ adb push path/to/form.xml /sdcard/odk/forms/form.xml
+  $ adb push local/path/to/form.xml /sdcard/odk/forms/form.xml
 
 .. note::
 
-  Path on the phone must include the file name and not just the folder name
+  The target path on the phone 
+  (the last part of the command)
+  must include the file name.
 
 .. _deleting-forms-with-adb:
 
 Deleting forms
 ~~~~~~~~~~~~~~~
 
-Forms can be deleted from :file:`sdcard/odk/forms` by running:
-
 .. code-block:: console
 
-  $ adb shell rm -d /sdcard/odk/forms/my_form.xml
+  $ adb shell rm -d /sdcard/odk/forms/form.xml
 
 .. _downloading-forms:
 
 Downloading forms to your computer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To download a completed form or form instance from the computer, run:
+To download a completed form or form instance from the computer:
 
 .. code-block:: console
 
-  $ adb pull /sdcard/odk/forms/my_form.xml
+  $ adb pull /sdcard/odk/forms/form.xml
 
+  
+.. _adb-dev-tasks:
+  
+Developer tasks and troubleshooting with ADB
+-----------------------------------------------
+  
 .. _downloading-database-with-adb:
 
-Downloading database
-~~~~~~~~~~~~~~~~~~~~~~
+Downloading Collect databases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Collect stores settings and form state information
+in several SQLite databases, 
+which you can pull onto your local computer.
 
-Developers might also need to check the entries in the database from the computer. In such case pull the database file from the SD card and use any **SQLite visualizer** to view it. To pull the database into the computer, run:
+To see a list of available databases:
 
 .. code-block:: console
   
-  $  adb -pull /sdcard/odk/database/database.name
+  $  adb ls /sdcard/odk/database/
```

Comment:
There's no `/odk/database` folder. The databases live in `/odk/metadata`. I'm not sure it's always been like that but I'm pretty sure...

Reply:
I should have double checked on the device, but I got this from the original wiki troubleshooting guide...